docs/_posts/examples/3400-01-01-setstyle.html

```diff
@@ -0,0 +1,45 @@
+---
+layout: example
+category: example
+title: Switch style
+description: Switch to a unique map style for a source.
```

Comment:
Why not just

 > Switch to another map style

I'm not sure what "unique"ness or sources have to do with this.